Day_5/environment.yml

```diff
@@ -0,0 +1,13 @@
+channels:
+  - bioconda
+  - conda-forge
+  - defaults
+  - ome
+  - joshmoore
```

Comment:
do we still need that channel?

Reply:
If you want the build of go-mc, then yes. We can migrate it to the org though if you'd like.

Reply:
I've begun the migration to https://github.com/ome/go-mc-feedstock

(Old repository will stay in place to not break existing notebooks)